src/wallet/rpcwallet.cpp

```diff
@@ -181,6 +181,8 @@ static void WalletTxToJSON(interfaces::Chain& chain, const CWalletTx& wtx, UniVa
             rbfStatus = "yes";
     }
     entry.pushKV("bip125-replaceable", rbfStatus);
+    if (confirms == 0)
```

Comment:
Needs braces or on a single line

Reply:
Ah right, I was just trying to make it match the surrounding code but I see from the style guide that you're correct. Will fix